style.css

```diff
@@ -409,33 +408,134 @@ form ::placeholder {
   line-height: 24px;
 }
 
-@media (min-width: 960px) {
+/* hamburger-menu */
+.humburger_menu {
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+}
+
+.mobile_menu {
+  top: 0;
+  left: -100%;
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  list-style: none;
+  background-color: rgba(96, 112, 255, 0.874);
+  padding-left: 28px;
+  padding-right: 28px;
+}
```

Comment:
- [ ] To match your design with the Figma design given to make the background of the mobile menu have a blurry look, it would be nice to add the blur effect to the styles. When I added this code `backdrop-filter: blur(2px);`, it achieved the look. 👍🏽

Reply:
thanks